.travis.yml

```diff
@@ -15,27 +15,14 @@ before_install:
 bundler_args: "--binstubs --standalone --without documentation --path ../bundle"
 script: "script/run_build"
 rvm:
-  - 1.8.7
-  - 1.9.2
-  - 1.9.3
-  - 2.0.0
-  - ree
   - rbx-3
-  - jruby-9.1.7.0 # pin JRuby to this until travis/rvm can install later versions
-  - jruby-head
-  - jruby-1.7
-env:
-  - JRUBY_OPTS='--dev'
 matrix:
   include:
-    - rvm: jruby-1.7
-      env: JRUBY_OPTS='--dev --1.8'
-    - rvm: 2.7.1
+    - rvm: 2.7.2
       env: DIFF_LCS_VERSION="~> 1.3.0"
-    - rvm: 2.7.1
+    - rvm: 2.7.2
```

Comment:
Don't need to bother changing these, they will be on github actions soon enough.